### ISSUE TITLE
Add an option to disable AppIDs renewal 

### ISSUE THREAD

**Issue by Bingo1upx** [altstoreio/AltStore#1282]
Let's imagine a situation in which a third-party application uses 7-8 AppIDs and takes up the entire free limit, but most of these AppIDs are practically useless and the application can safely work with 1-2 of these. I understand that deleting an AppID is not yet possible and you have to wait until it expires, but it would be cool to be able to turn off auto-renewal of unnecessary AppIDs without deleting the app

**Comment by shanegillio** [comment]
Have you used to option to remove extensions? If you have the IPA file on your device and tap the '+' icon in AltStore's My Apps tab, you should see a pop-up to remove extensions. If you do this, it may free up some AppIDs.

**Comment by Bingo1upx** [comment]
@shanegillio 
Yeah I know, but my app was installed long time ago without removing them and I want them to expire and not renew anymore without uninstalling/installing app
